code/main/edge/edgeRequestWorker.brs

```diff
@@ -132,16 +133,42 @@ function _adb_EdgeRequestWorker() as object
             return responseArray
         end function,
 
-        _processRequest: function(eventData as object, ecid as string, configId as string, requestId as string, path as string, edgeDomain = invalid as dynamic) as object
-            requestBody = m._createEdgeRequestBody(eventData, ecid)
+        _processRequest: function(eventData as object, ecid as string, datastreamId as string, requestId as string, path as string, meta = {} as object, edgeDomain = invalid as dynamic) as object
```

Comment:
I'm not familiar with the function's usage, but I prefer appending new default parameters to the end of the parameter list. Regarding "meta" do we require a default value considering we perform error checks within the function?